backend/src/main/java/quarano/department/web/TrackedCaseController.java

```diff
@@ -1,6 +1,7 @@
 package quarano.department.web;
 
-import static org.springframework.web.servlet.mvc.method.annotation.MvcUriComponentsBuilder.*;
+import static org.springframework.web.servlet.mvc.method.annotation.MvcUriComponentsBuilder.fromMethodCall;
+import static org.springframework.web.servlet.mvc.method.annotation.MvcUriComponentsBuilder.on;
```

Comment:
Please use the usual settings for imports: https://quarano.de/internal-test/api/docs/index.html#_import_order